modules/api-svc/src/lib/model/InboundTransfersModel.js

```diff
@@ -916,7 +1022,30 @@ class InboundTransfersModel {
             throw err;
         }
     }
+
+    // todo: combine with this._save
+    async saveFxState() { // fxQuote + fxTransfer
```

Comment:
Do we need a separate function for fxState?

Reply:
`_save()` uses different cache key based on _data.transferId_:
https://github.com/mojaloop/sdk-scheme-adapter/blob/feat/fx-impl-3618/modules/api-svc/src/lib/model/InboundTransfersModel.js#L1001

in FX payload we don't have _transferId f_ield